databricks/koalas/namespace.py

```diff
@@ -272,14 +293,36 @@ def read_csv(
 
         reader.options(**options)
 
-        if isinstance(names, str):
-            sdf = reader.schema(names).csv(path)
-        else:
-            sdf = reader.csv(path)
-            if header is None:
-                sdf = sdf.selectExpr(
-                    *["`%s` as `%s`" % (field.name, i) for i, field in enumerate(sdf.schema)]
+        try:
+            if isinstance(names, str):
+                sdf = reader.schema(names).csv(path)
+            else:
+                sdf = reader.csv(path)
+                if header is None:
+                    sdf = sdf.selectExpr(
+                        *["`%s` as `%s`" % (field.name, i) for i, field in enumerate(sdf.schema)]
+                    )
+        except Py4JJavaError as e:
+            with _fallback_to_pandas(e):
+                if quotechar is None:
+                    quotechar = '"'
+                pdf = pd.read_csv(
```

Comment:
Let's don't fallback to pandas' .. I thought about it multiple times and I think there's too many differences between Koalas and pandas. Users will expect to load files in a distributed manner. E.g., users might try to load `/tmp/abc` with default theme `fs.default.name` as `hdfs`. In the fallback, it will try to search `/tmp/abc` on local file name.